docs/core/porting/libraries.md

```diff
@@ -13,336 +13,183 @@ ms.assetid: a0fd860d-d6b6-4659-b325-8a6e6f5fa4a1
+To make code migration from .NET Framework easier, we've exposed some of the <xref:System.AppDomain> API surface in .NET Core. Some of the APIs function normally (for example, <xref:System.AppDomain.UnhandledException?displayProperty=fullName>), some of them do nothing (for example, <xref:System.AppDomain.SetCachePath%2A>), and some of them throw <xref:System.PlatformNotSupportedException> (for example, <xref:System.AppDomain.CreateDomain%2A>).
 
 ### Remoting
 
-For communication across processes, inter-process communication (IPC) mechanisms can be used as an alternative to Remoting, such as [Pipes](https://docs.microsoft.com/dotnet/core/api/system.io.pipes) or [Memory Mapped Files](https://docs.microsoft.com/dotnet/core/api/system.io.memorymappedfiles.memorymappedfile).
-
-Across machines, you can use a network based solution as an alternative, preferably a low-overhead plain text protocol such as HTTP. [KestrelHttpServer](https://github.com/aspnet/KestrelHttpServer), the web server used by ASP.NET Core, is an option here. Remote proxy generation via [Castle.Core](https://github.com/castleproject/Core) is also an option to consider.
-
-### Binary Serialization
-
-As an alternative to Binary Serialization, there are multiple different serialization technologies to choose. You should choose one that fits your goals for formatting and footprint. Popular choices include:
-
-* [JSON.NET](http://www.newtonsoft.com/json) for JSON
-* @System.Runtime.Serialization.DataContractSerializer for both XML and JSON
-* @System.Xml.Serialization.XmlSerializer for XML
-* [protobuf-net](https://github.com/mgravell/protobuf-net) for Protocol Buffers
+.NET Remoting, which is transparent remote procedure calls, was identified as a problematic architecture. It's used for cross-AppDomain communication, which is no longer supported. Also, Remoting requires runtime support, which is expensive to maintain. For these reasons, .NET Remoting isn't supported on .NET Core.
 
-Refer to the linked resources to learn about their benefits and choose the ones for your needs. There are many other serialization formats and technologies out there, many of which are open source.
+For communication across processes, consider inter-process communication (IPC) mechanisms as an alternative to Remoting, such as the <xref:System.IO.Pipes> or the <xref:System.IO.MemoryMappedFiles.MemoryMappedFile> class.
 
-### Sandboxes
+Across machines, use a network-based solution as an alternative. Preferably, use a low-overhead plain text protocol, such as HTTP. The [Kestrel web server](https://docs.microsoft.com/aspnet/core/fundamentals/servers/kestrel), the web server used by ASP.NET Core, is an option here. Consider <xref:System.Net.Sockets>. For more options, see [.NET Open Source Developer Projects: Messaging](https://github.com/Microsoft/dotnet/blob/master/dotnet-developer-projects.md#messaging).
```

Comment:
I'll expand the sentence to make it more "conversational," but that's a guidance they provided in the CoreFX topic, so I'm inclined to leave it here as an option for the reader to explore.